ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -390,6 +394,12 @@ tests:
     env:
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-gcp
+- as: e2e-aws-ovn-kubevirt
+  optional: true
+  run_if_changed: ^test/extended/kubevirt/.*
+  steps:
+    cluster_profile: aws
+    workflow: hypershift-kubevirt-conformance
```

Comment:
I think we need to add these environment variables to ensure we're testing with the right CNV version and operator options.
```
    env:
      CNV_PRERELEASE_VERSION: "4.14"
      ENABLE_HYPERSHIFT_OPERATOR_DEFAULTING_WEBHOOK: "true"
```

Reply:
OH! nice! I missed this pr about the CNV release detection.

The webhook isn't a big deal, but it is how we run the operator in production. Maybe we could move that envvar to our step-registry and default it to `true` rather than having to add it all over the place. 

Lets not worry about the webhook env var for now.